numpy/lib/index_tricks.py

```diff
@@ -1,8 +1,10 @@
 from __future__ import division, absolute_import, print_function
 
+import itertools
 import sys
 import math
 
+# Why are absolute imports not used here? or anywhere else in the numpy/lib??
```

Comment:
Do you mean relative imports? These _are_ absolute imports

Reply:
yeah, i guess i mean using the "dots"